src/basekernels/fbm.jl

```diff
@@ -17,7 +17,7 @@ struct FBMKernel{T<:Real} <: BaseKernel
     end
 end
 
-Base.show(io::IO, κ::FBMKernel) = print(io, "Fractional Brownian Motion Kernel (h = $(first(k.h)))")
+Base.show(io::IO, κ::FBMKernel) = print(io, "Fractional Brownian Motion Kernel (h = $(first(κ.h)))")
```

Comment:
Actually, interpolation is a bit unnecessary here (there's even a section in the [docs](https://docs.julialang.org/en/latest/manual/performance-tips/#Avoid-string-interpolation-for-I/O-1) that recommends not doing it):
```suggestion
Base.show(io::IO, κ::FBMKernel) = print(io, "Fractional Brownian Motion Kernel (h = ", first(κ.h), ")")
```
Of course, the same is true for all other implementations of `show`.